content/en/blog/releases/1-10-0/index.md

```diff
@@ -0,0 +1,163 @@
+
+```yaml
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: check-namespaces      
+spec:
+  validationFailureAction: Enforce
+  rules:
+  - name: call-extension
+    match:
+      any:
+      - resources:
+          kinds:
+          - ConfigMap
+    context:
+    - name: result
+      apiCall:
+        method: POST
+        data:
+        - key: somekey
+          value: "{{ somevariable }}"
+        service:
+          url: http://sample.myservice/someendpoint
+          caBundle: |-
+            -----BEGIN CERTIFICATE-----
+            <snip>
+            -----END CERTIFICATE-----            
+    validate:
+      message: "This shall not pass due to item {{ fookey}}"
+      deny:
+        conditions:
+          all:
+          - key: "{{ result.allowed }}"
+            operator: Equals
+            value: false
+```
+
+And, by the way, in addition to POST calls to external services, we've also enhanced the existing `apiCall` context variable to be able to POST to the Kubernetes API making it possible to [do things like](/policies/other/check-subjectaccessreview/check-subjectaccessreview/) pass a [SubjectAccessReview](https://kubernetes.io/docs/reference/kubernetes-api/authorization-resources/subject-access-review-v1/) which will make permissions assessments much easier.
+
+### Notary v2 Support
+
+The [Notary project](https://notaryproject.dev/) is another project, like Sigstore, aiming to solve software supply chain security through OCI image signing. Notary is currently in its second version and differentiates itself from Sigstore by using OCI artifacts to store image signatures. Although Kyverno has had support for verifying signatures and attestations from Sigstore's Cosign project for a while now, we wanted to add support for Notary v2 so that no matter what technology you use to sign your images, Kyverno will be there for you.
+
+Starting in Kyverno 1.10, we've added a new `type` field to verifyImages rules allowing you to specify the signatory used, either `NotaryV2` or `Cosign`. An example of what this looks like is shown below.
+
+```yaml
+apiVersion: kyverno.io/v2beta1
+kind: ClusterPolicy
+metadata:
+  name: check-image-notary
+spec:
+  validationFailureAction: Enforce
+  webhookTimeoutSeconds: 30
+  failurePolicy: Fail  
+  rules:
+    - name: verify-signature-notary
+      match:
+        any:
+        - resources:
+            kinds:
+              - Pod
+      verifyImages:
+      - type: NotaryV2
```

Comment:
Change to `Notary`. I will make the corresponding code change - it was previously done in a PR which was not merged to 1.10.